src/functionality.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-use-before-define */
+
+const removeTask = (task) => {
+  const tasksListEl = task.parentNode;
+  let tasksList = JSON.parse(localStorage.getItem('tasks'));
+  tasksListEl.removeChild(task);
+  const index = parseInt(task.id.slice(-1), 10);
+  tasksList = tasksList.filter((task) => task.index !== index);
+  const newTasksList = tasksList.map((task, i) => {
+    task.index = i + 1;
+    return task;
+  });
+  localStorage.setItem('tasks', JSON.stringify(newTasksList));
+  createTasksList();
+};
+
+const focus = (task) => {
+  task.classList.add('focus');
+  const deleteIcon = task.lastElementChild;
+  const dotsIcon = task.querySelector('.dots-icon');
+  dotsIcon.classList.remove('active');
+  deleteIcon.classList.add('active');
+};
+
+const blur = (task) => {
+  task.classList.remove('focus');
+  const deleteIcon = task.lastElementChild;
+  const dotsIcon = task.querySelector('.dots-icon');
+  dotsIcon.classList.add('active');
+  deleteIcon.classList.remove('active');
+};
+
+export const handleCheckbox = (checkbox) => {
+  const tasks = JSON.parse(localStorage.getItem('tasks'));
+  const index = parseInt(checkbox.id.slice(-1), 10);
+  tasks[index - 1].completed = checkbox.checked;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
```

Comment:
- [ ] Kindly add a new JavaScript file and import it as a module that will contain methods related to the status updates (completed: true/false). As stated in the requirements.